server/controllers/profilephoto.js

```diff
@@ -84,13 +80,8 @@ exports.downloadPhoto = asyncHandler(async (req, res) => {
                 if (error) {
                     res.json("Failed to retrieve an object: ");
                 } else {
-                    fs.writeFile("../client/src/Images/" + profile.photoUrl.substring(72), data.Body, (err) => {
-                        if (err)
-                            res.json('Error occured while writing file');
-                        else {
-                            res.json("File written successfully");
-                        }
-                    });
+                    res.setHeader("content-type", "some/type");

```

Comment:
```suggestion
                    res.setHeader("content-type", "image/jpeg");

```
or if you support multiple types you can use the version below for now:
```suggestion
                    res.setHeader("content-type", "image/*");

```